test/pt2_to_circle_test/test_pt2_to_circle.py

```diff
@@ -71,7 +71,12 @@ def convert_nnmodule_to_pt2(
     model: torch.nn.Module, example_inputs: tuple, pt2_model_path: str
 ):
     # Create .pt2 model
-    with torch.no_grad(), SuppressWarning(UserWarning, ".*quantize_per_tensor"):
```

Comment:
Not related with this PR but seems that `quantize_per_tensor` warning has been removed.